### PR TITLE
User story 4

### PR DESCRIPTION
**Pull request by kevxo**
Here I add the display of reviews in the shelter show page. Tests pass perfectly. 
Created the feature test in shelters/show_spec.

### CODE REVIEW

File: app/controllers/shelters_controller.rb [hunk 1]
```diff
@@ -23,6 +23,7 @@ def create
 
   def show
     @shelter = Shelter.find(params[:id])
+    @reviews = Review.all
```
Comment (jdcarey128) [inline app/controllers/shelters_controller.rb]: I think this should be: 
`@reviews = Review.where(shelter_id: @shelter.id)`
Otherwise, it will display all reviews for all shelters.

File: spec/features/shelters/show_spec.rb [hunk 2]
```diff
@@ -77,3 +77,51 @@
     end
   end
 end
+
+describe 'As a visitor' do
+  describe 'When I visit a shelters show page' do
+    it 'should show a list of reviews for that shelter' do
+      shelter1 = Shelter.create!(name: 'Dogs and Cats',
```
Comment (jdcarey128) [inline spec/features/shelters/show_spec.rb:84]: I think we should add an edge case shelter2 plus a review for shelter2 in this test to make sure that only the reviews for shelter1 are displayed on the shelter1 display page.

File: spec/features/shelters/show_spec.rb [hunk 3]
```diff
@@ -77,3 +77,51 @@
...
+      review2 = Review.create!(title: 'My thoughts',
+                               rating: 4,
+                               content: 'Great place for pets',
+                               picture: "https://upload.wikimedia.org/wikipedia/commons/4/4e/Tony_Romo_2015.jpg",
+                               shelter_id: shelter1.id,
+                               user_id: user2.id,
+                               name: user2.name)
+      visit "/shelters/#{shelter1.id}"
+
+      expect(page).to have_content(review1.title)
```
Comment (jdcarey128) [inline spec/features/shelters/show_spec.rb:134]: Once an edge case shelter and review are added, then you can add tests like: 
`expect(page).to_not have_content(review2.rating).to_s`
etc.

**Review by jdcarey128 — CHANGES_REQUESTED** [review]
A couple changes needed to account for edge cases.

**Review by jdcarey128 — APPROVED** [review]
